enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackupCommand.java

```diff
@@ -236,6 +236,7 @@ public void execute( String[] args ) throws IncorrectUsage, CommandFailed
         {
             try
             {
+                outsideWorld.stdOutLine( "Doing consistency check..." );
```

Comment:
I think they are both fine. I did `Doing` because it matches what we say for the import above:

> Doing full backup...